lib/Swagger/src/ContentEntity/Models/Universe.php

```diff
@@ -63,7 +65,8 @@ class Universe implements ArrayAccess
       */
     protected static $swaggerTypes = array(
         'id' => 'string',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "string" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)